lib/msprime.c

```diff
@@ -5893,6 +5893,24 @@ msp_activate_population(msp_t *self, int population_id)
     return ret;
 }
 
+/* Adds a population activation event.
+ * Note: we use the int32_t type here so we can be sure we're passing in
+ * arrays of the correct type from numpy where we have to specify the size.
+ */
+int MSP_WARN_UNUSED
+msp_add_activate_population(
```

Comment:
I'd change the name of this to ``msp_add_activate_population_event`` just for clarity.